cogs/birthday.py

```diff
@@ -7,12 +7,12 @@
 
 
 class Birthdays(commands.Cog):
-    DATEPATTERN: Pattern[str] = re.compile(r"(((0?[1-9])|([12][0-9]))\."  # 01.-29.
+    DATEPATTERN: Pattern[str] = re.compile(r"(((0?[1-9])|([12][0-9]))\.?"  # 01.-29.
```

Comment:
This is the "divider" dot. The "new" dot has to be in the line below.

Reply:
oops